parser/parser.go

```diff
@@ -322,9 +334,9 @@ func parseWithMetaCommentConfig(in []byte, c Config) ([]*ast.Node, error) {
 	if err != nil {
 		return nil, err
 	}
-	if p.log {
```

Comment:
Some logs, like this one, have V(2) by default, but others don't, so we need to somehow make the logger differentiate between "default verbosity" and "extra verbosity", such as having a boolean EnableExtraVerbosity.

Reply:
I didn't even notice that there were actually three log levels, not just two. There's Errorf, Infof, _and_ the verbose Infof. Quite confusing to be honest, but I'm also not used to glog :) Naively, I would call the third a Debugf in the interface.

I've taken you up on the extra boolean for the sake of unblocking the PR.

Reply:
Sorry, I was wrong, there is only one verbosity, the reason for the separate `if` statement is the performance:

https://pkg.go.dev/github.com/golang/glog#V

`The second form is shorter but the first is cheaper if logging is off because it does not evaluate its arguments.`

Please delete `ExtraVerbosity` and instead use something like `if p.config.log()` implemented as `c.Logger.IsVerbose()` (c *Config) implemented as `return bool(l.Verbose)` (l parserLog).

I've checked that this compiles using https://go.dev/play/p/e8pZ220VUGK.